README.md

```diff
@@ -43,6 +44,7 @@ Below you can find the information on the custom options to set up the configura
 
 | Option | Description | Type | Default | Required |
 |:---|:---|:---|:---|:---|
+| _AutoFlush_ | Defines if metrics are flushed synchronously or automatically in a background thread. | `boolean` | `false` | **NO** |
```

Comment:
Let's invert this configuration (say to `DisableAutoFlush`) so that the zero value (users not setting the config field) is a backwards compatible change.

I'd also document more clearly that disabling auto flush makes `FlushSize` and `FlushInterval` to be ignored, and that users must manually call one of the `FlushXXX` methods to flush the buffer.